src/bp/ui-studio/src/web/translations/fr.json

```diff
@@ -147,6 +147,7 @@
         "addCondition": "Ajouter une condition",
         "cantSeeActions": "Vous ne voyez pas votre action?",
         "chatbotSays": "L'assistant virtuel dit",
+        "chatbotPromptsUser": "L'agent conversationnel questionne l'utilisateur",
```

Comment:
That's long, i'm curious to see how this will fit on the node

Reply:
Probably very badly... We couldn't come up with something better :/